index.js

```diff
@@ -0,0 +1,18 @@
+require('dotenv').config();
+const path = require('path');
+const express = require('express');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+const proxyOptions = {
+  target: 'localhost:9000',
+  changeOrigin: true,
```

Comment:
I'm curious, what was your reason behind setting changeOrigin as true?

Reply:
The documentation on npm says that that's the way to make it work when you're running on a local machine, as I recall.

This configuration is not finished. I'm not even using the proxy properly yet, just loading in the other pages raw. I just left the options there that I knew I needed later.